samples/graph-appcatalog-lifecycle/csharp/README.md

```diff
@@ -80,7 +139,7 @@ This sample shows a AppCatalog bot and demonstrates teamsApp lifecycle in catalo
 ## Concepts introduced in this sample
 - List apps in catalog
 
-![image](https://user-images.githubusercontent.com/50989436/118778342-9ee83780-b8a7-11eb-93fc-96bf8448e8e0.png)
+![](AppCatalogSample/Images/1.PNG)
```

Comment:
Can we rename 1.png to some meaningful name?

Reply:
Updated to BotAction